service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/ManageOrdersControllerInitialSectionMidEventTest.java

```diff
@@ -79,8 +80,8 @@ void shouldNotPopulateHiddenFieldValuesWhenCreatingBlankOrderForTheCaseNotInClos
             .build();
 
         AboutToStartOrSubmitCallbackResponse response = postMidEvent(caseData, "initial-selection");
-
-        assertThat(response.getData()).doesNotContainKeys("orderTempQuestions", "issuingDetailsSectionSubHeader");
+        assertThat(response.getData()).doesNotContainKey("issuingDetailsSectionSubHeader");
+        assertThat(response.getData()).containsEntry("orderTempQuestions", null);
```

Comment:
Is this really what we want (instead of the "orderTempQuestions" not existing)?

Reply:
I think the above change was added due to us now storing the orderTempQuestions inside the ManageOrdersEventData which is causing orderTempQuestions to get a value of null when the CaseData object is built